scripts/measures/build-measures

```diff
@@ -1,11 +1,12 @@
 #!/usr/bin/env bash
 
-year=2018
+defaultPerformanceYear=2018
+performanceYear=${1:-$defaultPerformanceYear} # check if year provided on command line ($1)
```

Comment:
Since we won't every be rebuilding measures data for 2017, should this not be an argument and actually just default to 2018? Then we can remove the comment in Constants about editing this in two places.

Reply:
great point! ok

Reply:
hm... actually, I should keep the comment in Constants because its main use case is, when next year comes around, there's a second place where you should change it. I'll rename it from `default` to `current` though. 

removed it as an argument to build-measures!